### PR TITLE
bug - Add i18next to nunjucks constructor

### PR DESCRIPTION
**Pull request by KiranVarma02**
### Change description ###
bug - Add i18next to nunjucks constructor

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ *] No
```

### CODE REVIEW

File: .eslintignore [hunk 1]
```diff
@@ -8,4 +8,5 @@ coverage/*
 /src/main/public/
 # custom definition files
 /src/main/types/
-gulpfile.js
\ No newline at end of file
+gulpfile.js
+mochawesome-report
```
Comment (Jamesnelson117) [inline .eslintignore:12]: What about
coverage-report also?

File: src/main/modules/nunjucks/index.ts [hunk 2]
```diff
@@ -3,8 +3,9 @@ import * as express from 'express';
 import * as nunjucks from 'nunjucks';
 
 export class Nunjucks {
-  constructor(public developmentMode: boolean) {
+  constructor(public developmentMode: boolean, public i18next) {
```
Comment (tomaszpowroznik) [inline src/main/modules/nunjucks/index.ts:6]: can be strongly typed
import * as i18next from 'i18next';
i18next: i18next.i18n

Comment (Jamesnelson117) [inline src/main/modules/nunjucks/index.ts:6]: Are we just waiting for my PR for typing?

**Review by Jamesnelson117 — APPROVED** [review]
LGTM just a few things

File: src/main/modules/nunjucks/index.ts (hunk 2 above)
Comment (KiranVarma02) [reply src/main/modules/nunjucks/index.ts:6]: yes

File: .eslintignore (hunk 1 above)
Comment (KiranVarma02) [reply .eslintignore:12]: doesn't cause any issue at the moment .. does lint complain in your local for coverage-report?

Comment (Jamesnelson117) [reply .eslintignore:12]: I could of sworn it did but just tested again and no it does not. We can add it in if we find linter has problems with it.

**Pull request merged**